indispensable-asn1/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/ObjectIdentifier.kt

```diff
@@ -59,38 +55,40 @@ class ObjectIdentifier @Throws(Asn1Exception::class) private constructor(
      * for details.
      * Lazily evaluated.
      */
-    val bytes: ByteArray by if (bytes != null) lazyOf(bytes) else lazy {
-        this.nodes.toOidBytes()
+    val bytes: ByteArray by lazy {
+        bytes ?: nodes!!.toOidBytes()
     }
```

Comment:
didn't we talk about this already? why is this change back

Reply:
messed up the rebase